docker-compose/6.1.N-docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
 
     transform-router:
         mem_limit: 512m
-        image: quay.io/alfresco/alfresco-transform-router:1.0.2
+        image: quay.io/alfresco/alfresco-transform-router:1.3.2-A2
```

Comment:
Do we want to update 6.1.N at all, we normally only update latest and 6.2.N?

Reply:
reverted